fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/api/TenantStatus.java

```diff
@@ -10,9 +10,9 @@
  * Tenant Status Enumeration
  */
 public enum TenantStatus {
-    PROVISIONING,
-    FREE,
-    ALLOCATED,
-    FROZEN,
-    DROPPED
+    PROVISIONING, // in the process of adding the partitions to the schema
+    FREE,         // unused tenant, available for allocation (e.g. tenant pooling)
+    ALLOCATED,    // allocated and in use
+    FROZEN,       // about to be dropped
+    DROPPED       // all tenant resources have been release
```

Comment:
could we add a locked status to tenant which can help us to create a feature in the future to prevent access to the tenant temporarily?   or maybe we can re-use "FROZEN" for the purpose if it's not dedicated to dropped only.

Reply:
We can easily add a new status when we implement that new feature.